### PR TITLE
Use the CommandQuickAccess provider

### PR DESCRIPTION
**Pull request by lramos15**
Appears the `MenuRegistry` is missing editor commands for the command palette so commands like `Format Document` are not available. 

Maybe we should think about switching to the `CommandRegistry` at some point, but for now the command palette has nice names for common commads which makes finding what you want for development easy.

### CODE REVIEW

**Review by TylerLeonhardt — CHANGES_REQUESTED** [review]
Moving from platform to workbench means this is no longer available in Monaco standalone.

**Comment by lramos15** [comment]
> Moving from platform to workbench means this is no longer available in Monaco standalone.

This is a command I introduced to help me with some debugging related efforts. It was never intended to work in standalone monaco.

**Review by TylerLeonhardt — DISMISSED** [review]

**Comment by TylerLeonhardt** [comment]
FYI @bpasero would love your thoughts

**Review by TylerLeonhardt — APPROVED** [review]

File: src/vs/workbench/contrib/quickaccess/browser/commandsQuickAccess.ts [hunk 1]
```diff
@@ -127,6 +124,13 @@ export class CommandsQuickAccessProvider extends AbstractEditorCommandsQuickAcce
 		}));
 	}
 
+	public getCommandInfo(): Array<ICommandQuickPick> {
```
Comment (bpasero) [inline src/vs/workbench/contrib/quickaccess/browser/commandsQuickAccess.ts:127]:
```suggestion
	getCommandInfo(): Array<ICommandQuickPick> {
```

File: src/vs/workbench/contrib/quickaccess/browser/commandsQuickAccess.ts [hunk 2]
```diff
@@ -227,4 +231,15 @@ export class ClearCommandHistoryAction extends Action2 {
 	}
 }
 
+//#region --- Register a command to get all actions from the command palette
+CommandsRegistry.registerCommand('_getAllCommands', async function (accessor: ServicesAccessor) {
```
Comment (bpasero) [inline src/vs/workbench/contrib/quickaccess/browser/commandsQuickAccess.ts:235]: Why do we need this? It is somewhat ugly that we instantiate a UI component here only to get to the commands. Maybe the functionality for getting the commands should better be extracted to its own function or class?

**Review by bpasero — CHANGES_REQUESTED** [review]
I commented inline.

**Pull request merged**